packages/redux/src/brands/selectors.ts

```diff
@@ -63,6 +63,30 @@ export const areBrandsLoading = (
   hash = getBrandsHash(state),
 ) => (hash ? getIsLoading(state.brands as BrandsState)?.[hash] : undefined);
 
+/**
+ * Returns all brands from state.
+ *
+ * @param state - Application state.
+ *
+ * @returns Object with key values pairs representing brandId and brand properties.
+ */
+export const getBrands = (
+  state: StoreState,
+): Record<Brand['id'], Brand> | undefined => getEntities(state, 'brands');
+
+/**
+ * Returns a specific brand by its id.
+ *
+ * @param state   - Application state.
+ * @param brandId - Brand id.
+ *
+ * @returns Brand normalized.
+ */
+export const getBrand = (
+  state: StoreState,
+  brandId: Brand['id'],
+): Brand | undefined => getEntityById(state, 'brands', brandId);
```

Comment:
Do we need this `: Brand | undefined` return type?

Reply:
No, we don't. I was not looking at this when making these changes but I reviewed all the selectors I have worked with on this PR and removed any unnecessary typings. Can you take a look at it? Thanks!!